Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -56,6 +56,7 @@ static void InitAVCodec()
   if (first_run)
   {
     av_register_all();
+    avformat_network_init();
```

Comment:
@BhaaLseN that is exactly the point of this. I've tested streaming to twitch.tv (rtmp) and it appears to work just fine.